tests/components/zeroconf/test_init.py

```diff
@@ -78,11 +82,15 @@ async def test_setup(hass, mock_zeroconf):
         expected_flow_calls += len(matching_components)
     assert len(mock_config_flow.mock_calls) == expected_flow_calls
 
+    # Test instance is set.
+    assert "zeroconf" in hass.data
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5518609)